services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowPingSpec.groovy

```diff
@@ -239,6 +242,45 @@ class FlowPingSpec extends HealthCheckSpecification {
         }
     }
 
+    @Tidy
+    @Tags(HARDWARE)
+    def "Flow ping can detect a broken path for a vxlan flow on an intermediate switch"() {
+        given: "A vxlan flow with intermediate switch(es)"
+        def switchPair = topologyHelper.getAllNotNeighboringSwitchPairs().find { swP ->
+            swP.paths.findAll { path ->
+                pathHelper.getInvolvedSwitches(path).every { it.noviflow  && !it.wb5164 }
```

Comment:
You must select by looking at switch properties. This search does not guarantee that switches are vxlan-enabled

Reply:
I forgot about that, thx.
 fixed